src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentThreadPartial.cshtml

```diff
@@ -21,12 +21,14 @@
                             <div class="comment-actions">
                                 <form data-post-update="comments" asp-controller="Comments" asp-action="ToggleUpvote" method="post" asp-route-reviewId="@Model.ReviewId">
                                     <input type="hidden" name="commentId" value="@comment.CommentId" />
-                                    <button type="submit" class="btn btn-light btn-sm btn-upvote" title="@string.Join(", ", comment.Upvotes) " active-if="@comment.Upvotes.Contains(User.GetGitHubLogin())">
-                                        @if (comment.Upvotes.Any())
-                                        {
-                                            @comment.Upvotes.Count
+                                    <button type="submit" class="btn btn-light btn-sm btn-upvote" title="@string.Join(", ", comment.Upvotes)" active-if="@comment.Upvotes.Contains(User.GetGitHubLogin())">
+                                        @{
+                                            if (comment.Upvotes.Any())
+                                            {
+                                                @comment.Upvotes.Count
+                                            }
+                                            <span active-if="@comment.Upvotes.Contains(User.GetGitHubLogin())">👍</span>
```

Comment:
I think the condition is wrong here. It should be `comment.Upvotes.Any()`.

Reply:
I was a bit confused then based on earlier suggestion of 3 different format based on your previous comment. We want to show upvote as active when only current user upvoted it. Let me check.
```
For upvotes I think the right thing is:
Person that upvoted - active button, not grey
Others - not active button, not grey
```